views/layouts/default.hbs

```diff
@@ -42,21 +42,21 @@
     <footer id="footer">
       <div class="section-wrapper">
         <div class="third">
-          <a class="footer-logo-wrapper" href="https://www.mozilla.org" target="_blank" rel="noopener"><img src="/img/moz-logo-bw-rgb.svg" alt="Mozilla Logo" /></a>
+          <a class="footer-logo-wrapper" href="https://www.mozilla.org?utm_source={{ SERVER_URL }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=footer-link" target="_blank" rel="noopener"><img src="/img/moz-logo-bw-rgb.svg" alt="Mozilla Logo" /></a>
         </div>
         <ul class="third">
           <li>Support</li>
           <li><a href="https://qsurvey.mozilla.com/s3/Firefox-Monitor-Feedback" target="_blank" rel="noopener">Give Feedback</a></li>
-          <li><a href="https://www.mozilla.org/about/legal/" target="_blank" rel="noopener">Terms</a></li>
-          <li><a href="https://www.mozilla.org/privacy/websites/" target="_blank" rel="noopener">Privacy</a></li>
+          <li><a href="https://www.mozilla.org/about/legal/?utm_source={{ SERVER_URL }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=footer-link" target="_blank" rel="noopener">Terms</a></li>
```

Comment:
Do we want the full `{{ SERVER_URL }}` here, or should we strip the "https://" prefix? Does it need to be URL-encoded at all?

Same as above, should we convert it into some handy HBS variable that we can recycle a bit easier:

```hbs
<a href="https://www.mozilla.org/about/legal/?{{ utm_site_footer }}" target="_blank" rel="noopener">...</a>
```

Reply:
@peterGerman also suggests stripping to the hostname. I'm adding a new `UTM_SOURCE` app local that will take care of it.